pymongo/aggregation.py

```diff
@@ -46,6 +47,8 @@ def __init__(self, target, cursor_class, pipeline, options,
         if let:
             common.validate_is_mapping("let", let)
             options["let"] = let
+        if comment:
```

Comment:
Here and everywhere else this needs to be `if comment is not None` otherwise we won't send falsey things as comments,  eg comment=False, comment={}, comment=0.

Reply:
Done.